test/api/workflows_format_2/converter.py

```diff
@@ -1,14 +1,13 @@
-import os
-import sys
+"""Functionality for converting a Format 2 workflow into a standard Galaxy workflow."""
+from __future__ import print_function
 
-import yaml
+from collections import OrderedDict
```

Comment:
Can we move this discussion to https://github.com/galaxyproject/galaxy/issues/2300 - I'm happy to discuss this but since Galaxy doesn't currently prescribe an import order or offer any linting to do this it would seem inappropriate to delay this PR on import order discussions or concerns.